### DISCUSSION TITLE
Have any example of navbar with effect ?

### DISCUSSION THREAD

**Discussion by ecatugy** [mantinedev/mantine#3552] [Q&A]
Has any example when click in burger in navbar close menu with effect like menu in antd ?

![image](https://user-images.githubusercontent.com/22201052/218844718-aba519b6-087a-4b66-97f1-7a88245cde7c.png)

**Comment by rtivital** [comment]
You can find examples of prebuilt components on Mantine UI website – https://ui.mantine.dev/

**Reply by ecatugy** [reply]
There are good examples, but not with the burger button closing the NavBar with the overlay effect.

**Reply by rtivital** [reply]
See Popover component

**Reply by ecatugy** [reply]
It's totally different from what I need, thanks anyway.

**Reply by shurshilov** [reply]
I'm facing the same problem.When you start writing, you need a menu in the navbar, for example, it is in antd. In the mantivnedev example, there are similar components NavbarGroup and NavbarLink, but they are difficult to implement and although they look great. I guess the navbarMenu component is missing, which had a user-friendly interface for creating and looked about the same as the menu in mantinedev. 3 levels of nesting is enough, although I've done unlimited.
Tell me, could you implement such a component?

**Reply by shurshilov** [reply]
And another cool effect would be when, when clicking on the burger, the menu was not just hidden, but displayed only icons in an enlarged size. It is clear that all this can be done by yourself. But it seems to me that this is the template code that many people write, simplify the task for us)))

**Reply by shurshilov** [reply]
I do some like this 
![image](https://github.com/mantinedev/mantine/assets/11828278/c56e64c2-c86d-4e1d-88c6-f1b2068854eb)
![image](https://github.com/mantinedev/mantine/assets/11828278/8f8fdd43-d5fd-49a3-b343-4e0331443cb0)

if any interest i can share code
